examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/ChipClient.kt

```diff
@@ -41,7 +41,7 @@ object ChipClient {
   private lateinit var chipDeviceController: ChipDeviceController
   private lateinit var androidPlatform: AndroidChipPlatform
   /* 0xFFF4 is a test vendor ID, replace with your assigned company ID */
-  private const val VENDOR_ID = 0xFFF4
```

Comment:
should we consider to add UI to set this vendor ID so companies can quicky play around the vendorId for testing purpose?

Reply:
This value must be the VendorID of the controller company. This was implemented like this because I thought that the same value as the value set in the code below should be set. Wouldn’t there be a separate UI?
https://github.com/project-chip/connectedhomeip/blob/f0e6fc99535f63d6e71e6c83e931d86d1b267987/examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/ChipClient.kt#L51

Reply:
yep, maybe separate UI would be cool, but the default value would be 0xFFF4. Assume developer wanna firstly use android chip-tool to exercise their company's product, they may wanna setup the UI as his company's vendor Id, and try to simulate as much as they can.

Reply:
I added some UI (EditText) for changable vendorID.